doc_source/ContainerInsights-Prometheus-Sample-Workloads-appmesh-envoy.md

```diff
@@ -12,28 +12,38 @@ You can set up Container Insights FluentD to send App Mesh Envoy access logs to
 
    1. To use the App Mesh CLI to enable or update the Envoy access logs, use the `update-virtual-node` command\. For more information, see [update\-virtual\-node](https://docs.aws.amazon.com/cli/latest/reference/appmesh/update-virtual-node.html)\.
```

Comment:
The prometheus integration is really only within kubernetes right? In which case we'd want to remove the console/CLI stuff since the controller won't like it?

That being said, could also just link to this for enabling access logging https://docs.aws.amazon.com/app-mesh/latest/userguide/observability.html#envoy-logs

Reply:
Good points, I will remove. I left it incase another guide leads to this generic page